packages/apps/immutable-records/next-env.d.ts

```diff
@@ -0,0 +1,5 @@
+/// <reference types="next" />
+/// <reference types="next/image-types/global" />
+
+// NOTE: This file should not be edited
```

Comment:
what is this file for?

Reply:
I need to remove this! It must have been a local file